StripeCryptoOnramp/StripeCryptoOnramp/Source/Internal/API Bindings/Credentials.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Credentials.swift
```

Comment:
I moved this out `CustomerRequest` since now both of our requests make use of it.